syntax = "proto3";

package Data;

message UserCreate
{
	string user_name = 1;//用户ID
	string user_pwd = 2;//用户密码
}
